src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -3194,151 +3213,165 @@ namespace HVACVariableRefrigerantFlow {
 
             if (UtilityRoutines::SameString(cAlphaArgs(6), "BlowThrough")) VRFTU(VRFTUNum).FanPlace = DataHVACGlobals::BlowThru;
             if (UtilityRoutines::SameString(cAlphaArgs(6), "DrawThrough")) VRFTU(VRFTUNum).FanPlace = DataHVACGlobals::DrawThru;
-            if (VRFTU(VRFTUNum).FanPlace == 0) {
-                ShowSevereError(cCurrentModuleObject + " = " + VRFTU(VRFTUNum).Name);
-                ShowContinueError("Illegal " + cAlphaFieldNames(6) + " = " + cAlphaArgs(6));
-                ErrorsFound = true;
-            }
 
-            // Get fan data
-            FanType = cAlphaArgs(7);
-            FanName = cAlphaArgs(8);
-            if (UtilityRoutines::SameString(FanType, "Fan:SystemModel")) {
-                if (!HVACFan::checkIfFanNameIsAFanSystem(FanName)) {
-                    ErrorsFound = true;
```

Comment:
Fan no longer required for air loop and OS system TUs. Fan is required for zone equipment and will be checked for later.